### PR TITLE
[MRG] Fixing typos

### PR DESCRIPTION
**Pull request by betatim**
This PR fixes several typos in our website.

### CODE REVIEW

File: site/content/english/integration.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ og_image: og/og-skribble-identification.jpg
 {{< content class="no-bottom-padding" >}}
 {{% heading %}}
 # Skribble Integration
-With Skribble, you choose a cloud platform that combines
+With Skribble, you choose a platform that combines
```
Comment (betatim) [inline site/content/english/integration.md]: Removes the word "cloud".

File: site/content/french/integration.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ og_image: og/og-skribble-identification.jpg
 {{< content >}}
 {{% heading %}}
 # Intégration de Skribble
-Avec Skribble, vous choisissez une plate-forme de cloud 
+Avec Skribble, vous choisissez une plate-forme
```
Comment (betatim) [inline site/content/french/integration.md]: Removes the word cloud

File: site/content/german/integration.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ og_image: og/og-skribble-identification.jpg
 {{< content >}}
 {{% heading %}}
 # Skribble Integration
-Mit Skribble wählst du eine Cloud-Plattform, 
+Mit Skribble wählst du eine Plattform,
```
Comment (betatim) [inline site/content/german/integration.md]: Removes the word cloud

**Comment by michelfernandez** [comment]
I've gone over all the corrections. They all make sense and I'd leave out the one about the cloud. We had discussed calling it a platform anyway, but we haven't implemented it yet.

File: site/content/german/integration.md (hunk 3 above)
Comment (betatim) [reply site/content/german/integration.md]:
```suggestion
Mit Skribble wählst du eine Cloud-Plattform, 
```

File: site/content/french/integration.md (hunk 2 above)
Comment (betatim) [reply site/content/french/integration.md]:
```suggestion
Avec Skribble, vous choisissez une plate-forme de cloud
```

File: site/content/english/integration.md (hunk 1 above)
Comment (betatim) [reply site/content/english/integration.md]:
```suggestion
With Skribble, you choose a cloud platform that combines
```

**Comment by fursinger** [comment]
Please make sure to update https://app.asana.com/0/1160025184996537/1160563380222793 accordingly. I just stumbled across it because I had merge conflicts between master and #283.

**Pull request merged**